### PR TITLE
KEYCLOAK-5543 - Edited QS readme

### PR DESCRIPTION
**Pull request by Pepo48**
To preserve consistency, alternative client creation option is mentioned.

### CODE REVIEW

**Comment by abstractj** [comment]
@matthewhelmke would you like to review?

File: service-jee-jaxrs/README.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ different host. To do this edit `keycloak.json` and add:
 }
 ````
 
+As an alternative you can create the client by importing the file [client-import.json](config/client-import.json) and
+copying [config/keycloak-example.json](config/keycloak-example.json) to `config/keycloak.json`.
```
Comment (matthewhelmke) [inline service-jee-jaxrs/README.md]: As an alternative, you can create the client by importing the [client-import.json](config/client-import.json) file and
copying [config/keycloak-example.json](config/keycloak-example.json) to `config/keycloak.json`.

**Comment by abstractj** [comment]
@Pepo48 could you please update based on Matthew's comments?

**Comment by Pepo48** [comment]
@abstractj according to suggested, it seems that similar edit could be done in another 4 QSs, at least (I have also just copied this sentence from another QS). Also, because of fact that this PR is an "orphan" (I have deleted an original fork), I would suggest close this one and create new, with @matthewhelmke  change in every other QS readme that contains the same sentence. Do you agree?

**Comment by abstractj** [comment]
@Pepo48 +1 closing it, thank you.

**Pull request closed without merging**